db/migrate/20170810030748_add_avatar_to_members.rb

```diff
@@ -1,3 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Do we need a separate PR for Rubocop cleanup? Are these updates a blocker for your PR?